src/LibChorus/merge/ChangeReport.cs

```diff
@@ -0,0 +1,76 @@
+﻿// Copyright (c) 2018 SIL International
+// This software is licensed under the MIT License (http://opensource.org/licenses/MIT)
+using System;
+using Chorus.VcsDrivers.Mercurial;
+using Palaso.Providers;
+
+namespace Chorus.merge
+{
+	public abstract class ChangeReport : IChangeReport
+	{
+		public FileInRevision ChildFileInRevision { get; private set; }
+		public FileInRevision ParentFileInRevision { get; private set; }
+		protected Guid _guid = GuidProvider.Current.NewGuid();
+
+		//        public override bool Equals(object obj)
```

Comment:
I'm not a fan of committing commented out code, even though this was just merged. If there isn't an immediate plan for it we can just remove it.

Reply:
Good point. Done.